indico/modules/users/controllers.py

```diff
@@ -354,6 +354,11 @@ def _process(self):
         return WPUser.render_template('preferences.html', 'preferences', user=self.user, form=form)
 
 
+class RHUserPreferencesMarkdownAPI(RHUserBase):
+    def _process(self):
+        return jsonify(session.user.settings.get('use_markdown_for_minutes'))
```

Comment:
```suggestion
        return jsonify(self.user.settings.get('use_markdown_for_minutes'))
```

not really relevant here but all these user endpoints are defined with `/user/<id>/...` as well in the blueprint (check the `with` block there), so they need to use the correct user in case a user id it specified